test/ufds.test.js

```diff
@@ -22,7 +20,12 @@ var UFDS = require('../lib/index');
 
 // --- Globals
 
-var UFDS_URL = process.env.UFDS_URL || 'ldaps://10.99.99.18';
+assert.string(process.env.UFDS_IP, 'UFDS_IP envvar');
+assert.string(process.env.UFDS_LDAP_ROOT_PASSWORD,
+    'UFDS_LDAP_ROOT_PASSWORD envvar');
+
+var UFDS_URL = 'ldaps://' + process.env.UFDS_IP;
+var UFDS_PASSWORD = process.env.UFDS_LDAP_ROOT_PASSWORD
```

Comment:
Same again.

Reply:
Sorry about that. See https://github.com/joyent/node-ufds/pull/6